.travis.yml

```
@@ -1,19 +1,16 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
+  - "14"
 
 cache:
   directories:
     - node_modules
 
 install:
-  - npm install -g yarn
-  - yarn install
+  - npm install gatsby@2.20.36
```

```suggestion
  - npm install
```

You actually don't need to set the gatsby version like this. If you just run `npm install`, it'll install all of the packages and versions specified in `package.json`.